tests/h/auth/tokens_test.py

```diff
@@ -36,100 +35,79 @@ def test_token_with_past_expiry_is_not_valid(self):
 
 VALID_TOKEN_EXAMPLES = [
     # Valid
-    lambda a, k: jwt.encode({'aud': a, 'exp': _seconds_from_now(3600)},
```

Comment:
I've just checked the client code and it looks like we don't ever look at the `aud` property, so this should be fine.